TESTING_AGAINST_AWS.md

```diff
@@ -0,0 +1,93 @@
+# Testing Against AWS - Integration Testing
+
+## General Approach
+
+In general, we use Terraform to setup test objects in AWS, then run a predefined set of InSpec controls against it (which should all pass); finally we tear down the the environment using Terraform.
+
+We also use the AWS CLI credentials system to manage credentials.
+
+### Installing Terraform
+
+Download [Terraform](https://www.terraform.io/downloads.html).  We currently require at least v0.10 . You may also consider using [tfenv](https://github.com/kamatama41/tfenv), which allows you to install and choose from multiple versions.
+
+### Installing AWS CLI
+
+Install the [AWS CLI](http://docs.aws.amazon.com/cli/latest/userguide/installing.html). We will store profiles for testing in the `~/.aws/credentials` file.
+
+## Limitations
+
+There are some things that we can't (or very much shouldn't) do via Terraform - like manipulating the root account MFA settings.
+
+Also, there are some singleton resources (such as the default VPC, or Config status) that we should not manipulate without consequences.
+
+## Current Solution
+
+We create two AWS accounts, each dedicated to the task of integration testing inspec-aws.  Both accounts will be manually configured (to cover the gap between what we want to examine, and what test fixtures Terraform can set up).  Between the two of them, we want to have a positive and a negative compliance check for each test point.
```

Comment:
Could we make the bit in `()` part of the sentence? Reads better to me that way.